Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function (grunt) {
```

Comment:
Excellent point.  One thing I'll make a point of doing before MAGStock (and all of our other events) is to get my hands on one of the older laptops we use and test out Uber on them to make sure they'll be okay.  If they're noticeably sluggish then that means we definitely need to make changes.  Once our pages perform well on those laptops we know we've made enough optimizations.  (Further optimizations beyond that would obviously be fine, just not required.)